doc/crdt-details.md

```diff
@@ -303,7 +332,8 @@ Takes two `Subset`s and produces a new `Subset` of the same string where each ch
 
 `Subset::transform_expand` takes a `Subset` and another "transform" `Subset` and transforms the first `Subset` through the coordinate transform represented by the "transform". Now what does this mean:
 
-`Revision`s are never modified even when the union string expands with newly inserted characters. We can deal with this by treating `Subset`s of inserted characters as coordinate transforms. The only difference to the union string is the inserted characters, so if we can modify the coordinates of a `Subset` from being based on one union string to another, we can work with edits from multiple `Revision`s together.
+
+`Revision`s are never modified, and their `Edit`s always refer to the union string that existed when they were created. For operations on multiple `Revision`s, have to be able to map coordinates in one's union string to coordinates in another's. We can deal with this by treating `Subset`s of inserted characters as coordinate transforms. The only difference to the union string is the inserted characters, so if we can modify the coordinates of a `Subset` from being based on one union string to another, we can work with edits from multiple `Revision`s together.
```

Comment:
second last sentence: I might try "**As** the only difference [...] characters, ~so~ if [...]"

and later on, maybe "if we can **map** ~modify~ the coordinates of a `Subset` from ~being based on~ one union string to another,"